doc/devel/testing.rst

```diff
@@ -33,6 +33,22 @@ Optionally you can install:
 
   - `pep8 <http://pep8.readthedocs.org/en/latest>`_ to test coding standards
 
+Build matplotlib for image comparison tests
```

Comment:
The tense here is different from most of the rest of the file (though it's not all totally the same.)